app/src/forms/form/exportService.js

```diff
@@ -62,44 +62,63 @@ const service = {
     return `${form.snake()}_${type}.${format}`.toLowerCase();
   },
 
-  _submissionsColumns: (params = {}) => {
+  _submissionsColumns: (form, params = {}) => {
     if (params.columns) {
       return params.columns;
     }
-    return ['confirmationId', 'formName', 'version', 'createdAt', 'fullName', 'username', 'email', 'submission'];
+    // Custom columns not defined - return default column selection behavior
+    let columns = [
+      'confirmationId',
+      'formName',
+      'version',
+      'createdAt',
+      'fullName',
+      'username',
+      'email'
+    ];
+    // if form has 'status updates' enabled in the form settings include these in export
+    if (form.enableStatusUpdates) {
+      columns = columns.concat(['status', 'assignee', 'assigneeEmail']);
+    }
+    // and join the submission data
+    return columns.concat(['submission']);
   },
 
-  _getForm: async (formId) => {
+  _getForm: (formId) => {
     return Form.query().findById(formId).throwIfNotFound();
   },
 
-  _getData: async (exportType, formId, params = {}, currentUser) => {
+  _getData: (exportType, form, params = {}, currentUser) => {
     if (EXPORT_TYPES.submissions === exportType) {
-      return await service._getSubmissions(formId, params, currentUser);
+      return service._getSubmissions(form, params, currentUser);
     }
     return {};
   },
 
   _formatData: async (exportFormat, exportType, form, data = {}) => {
```

Comment:
Good candidate for unit testing. Does array object assignment, conditional logic depending on params, and calls out to other mockable functions.
Actually plenty of unit testing possibility in this exportService and there isn't any test file for it yet so a good thing to backfill at some point.